py3-graph-tool.yaml

```diff
@@ -0,0 +1,184 @@
+package:
+  name: py3-graph-tool
+  version: 2.97
+  epoch: 0
+  description: graph-tool is an efficient Python module for manipulation and statistical analysis of graphs (a.k.a. networks).
+  resources:
+    cpu: 65
+    memory: 128Gi
+  copyright:
+    - license: GPL-3.0-only
+
+vars:
+  pypi-package: graph-tool
+  import: graph_tool
+
+data:
+  # TODO: add other supported versions
```

Comment:
What about other Python versions @kranurag7? Any reason to not add them here?

Reply:
Jamon, the reason yesterday was that it takes ~45 minutes to build. 

the reason today is that I'm running into linker issues with older versions.

```bash
/usr/lib/gcc/x86_64-pc-linux-gnu/14/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lboost_python312: No such file or directory
```
I'm still looking into this as of now.